From 380454dd8a29e5aa15cbad98c05ff3a4472f8a14 Mon Sep 17 00:00:00 2001
From: Sandor Yu <R01008@freescale.com>
Date: Mon, 13 Jul 2015 16:22:53 +0800
Subject: [PATCH 0911/1221] MLK-10732-2: mxsfb: Add FB_VMODE_YWRAP support

Add FB_VMODE_YWRAP support in mxsfb driver.

Signed-off-by: Sandor Yu <R01008@freescale.com>
[zou:Original patch taken from
git.freescale.com/imx/fsl-arm-yocto-bsp.git-b imx-3.14.52-1.1.0_ga]
Signed-off-by: zou cao <cao.zou@windriver.com>
---
 drivers/video/mxsfb.c |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/drivers/video/mxsfb.c b/drivers/video/mxsfb.c
index 72378cd..e9c61df 100644
--- a/drivers/video/mxsfb.c
+++ b/drivers/video/mxsfb.c
@@ -1105,6 +1105,7 @@ static int mxsfb_restore_mode(struct mxsfb_info *host)
 
 	line_count = fb_info->fix.smem_len / fb_info->fix.line_length;
 	fb_info->fix.ypanstep = 1;
+	fb_info->fix.ywrapstep = 1;
 
 	host->enabled = 1;
 
@@ -1223,6 +1224,7 @@ static int mxsfb_init_fbinfo(struct mxsfb_info *host)
 	fb_info->flags = FBINFO_FLAG_DEFAULT | FBINFO_READS_FAST;
 	fb_info->fix.type = FB_TYPE_PACKED_PIXELS;
 	fb_info->fix.ypanstep = 1;
+	fb_info->fix.ywrapstep = 1;
 	fb_info->fix.visual = FB_VISUAL_TRUECOLOR,
 	fb_info->fix.accel = FB_ACCEL_NONE;
 
-- 
1.7.5.4

